content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -254,15 +254,20 @@ Windows has strict compatibility rules, where the host OS version must match the
 * The existing node problem detector is Linux-only and requires privileged containers. In general, we don't expect this to be used on Windows because privileged containers are not supported
 * Not all features of shared namespaces are supported (see API section for more details)
 
-##### Memory Reservations and Handling
+##### Resource Reservations and Handling
 
+###### Memory Reservations
 Windows does not have an out-of-memory process killer as Linux does. Windows always treats all user-mode memory allocations as virtual, and pagefiles are mandatory. The net effect is that Windows won't reach out of memory conditions the same way Linux does, and processes page to disk instead of being subject to out of memory (OOM) termination. If memory is over-provisioned and all physical memory is exhausted, then paging can slow down performance.
 
 Keeping memory usage within reasonable bounds is possible with a two-step process. First, use the kubelet parameters `--kubelet-reserve` and/or `--system-reserve` to account for memory usage on the node (outside of containers). This reduces [NodeAllocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)). As you deploy workloads, use resource limits (must set only limits or limits must equal requests) on containers. This also subtracts from NodeAllocatable and prevents the scheduler from adding more pods once a node is full.
```

Comment:
@marosset and I also discussed pulling and a bigger emphasis on the `(limits must equal requests)`  by pulling it out into a `Note` section.  We see a lot of folks over subscribe their clusters and get into a bad state due to this. By making it more prominent in the documentation it would help. Seems like this would be a good PR to do that in.

Reply:
I am fine doing that but over subscription is a problem with Linux too are we recommending not to do it at all for Windows?

Reply:
For memory we are recommending that it is not over subscribed since there are not things like OOM.  

For CPU, I think the idea was to provide some guidance in that direction since we are seeing the nodes become unresponsive in the over subscription scenarios.  I am not familiar enough what happens on the Linux side.